docs/release-branch-backporting.md

```diff
@@ -0,0 +1,28 @@
+# Release Branch Backporting Policy
+
+Bug fixes are eligible to be backported from the master branch to any previous
+release branch. The following criteria must be met before a backport can be
+considered. It is the reviewer's and approver's responsibility to uphold this
+policy.
+
+- **Bug Fix Only:** The backport must be a bug fix and the bug fix must be
```

Comment:
There were a lot of discussions in the past regarding to infra-related changes: It would be very helpful for infra-work and test-stabiliziation PRs can just be done. Without creating somewhere extra issues (if on github, there is no clear opinion if PRs need an issue, I would prefer not, since it is close enough together).

Reply:
> There were a lot of discussions in the past regarding to infra-related changes

I'm going to explicitly call out infra changes as being welcome for backporting just to make sure there's no confusion later on. 

> Without creating somewhere extra issues

yep, makes sense. I think i have reflected that in the doc. I'm not of the opinion that we have to have issues created somewhere for every PR we consider backporting. I think it's enough that we have a policy of ensuring that a reviewer can accurately determine using information within the description (whether links or a text explanation) the purpose of the backport.